az-core/src/test/java/azkaban/utils/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+* Copyright 2018 LinkedIn Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the “License”); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an “AS IS” BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+package azkaban.utils;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.zip.ZipFile;
+import org.junit.Test;
+
+/**
+ * Test class for azkaban.utils.Utils
+ */
+public class UtilsTest {
+
+  private static final String INVALID_ZIP_FILE = "zip-slip.zip";
```

Comment:
Does this resource need to be included in this PR as well?

Reply:
My bad, I forgot to include in the commit, thanks!

Reply:
Good question.

Reply:
I created the zip in the code using ZipOutputStream instead of uploading the actual zip. This would be clearer to the users what the content would look in the zip.